lang/python/python-intelhex/Makefile

```diff
@@ -0,0 +1,41 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-intelhex
+PKG_VERSION:=2.2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/i/intelhex/
+PKG_SOURCE:=intelhex-$(PKG_VERSION).tar.gz
+PKG_HASH:=009d8511e0d50639230c39af9607deee771cf026f67ef7507a8c3fd4fa927832
+
+PKG_MAINTAINER:=Karel Kočí <cynerd@email.cz>
+PKG_LICENSE:=BSD
```

Comment:
Change to BSD-3-Clause

Add PKG_LICENSE_FILES:=LICENSE.txt

Reply:
Added

Reply:
Change to BSD-3-Clause please. BSD is not a proper SPDX tag.